snaps/index.mdx

```diff
@@ -129,6 +116,35 @@ the canary distribution of MetaMask:
   ]}
 />
 
+## New to Snaps?
+
+If you're new to Snaps, get started learning with the following topics:
+
+<CardList
+  items={[
+    {
+      href: "concepts/overview",
+      title: "Snaps overview",
+      description: "See a high-level, technical overview of the Snaps system.",
```

Comment:
```suggestion
      description: "Review a high-level, technical overview of the Snaps system.",
```

Reply:
"Review" connotes viewing something _again_. I think "see," "view," or "read" are the simplest verbs to use in these instances. Since we intend to have a visual diagram on this page, "read" isn't the best. And since the sentence already contains "overview," "view" might be too similar.